packages/frontend/src/schema.ts

```diff
@@ -0,0 +1,172 @@
+/**
+ * This file was auto-generated by openapi-typescript.
+ * Do not make direct changes to the file.
+ */
+
+export interface paths {
+  "/api/v1/opportunity": {
+    get: operations["OpportunityController_findAll"];
+    post: operations["OpportunityController_create"];
+  };
+  "/api/v1/opportunity/{id}": {
+    get: operations["OpportunityController_findOne"];
+    put: operations["OpportunityController_update"];
+    delete: operations["OpportunityController_delete"];
+  };
+}
+
+export interface components {
+  schemas: {
+    OpportunitySummaryResponseDto: {
+      title: string;
+      contactName: string;
+      requiredPeopleCount: number;
+      /** @example 1666666666 */
+      startTime: number;
+      /** @example 1666666666 */
+      endTime: number;
+      /** @example 1666666666 */
+      postedTime: number;
+      opportunityId: string;
+    };
+    OpportunityCreateDto: {
+      title: string;
+      contactName: string;
+      requiredPeopleCount: number;
+      /** @example 1666666666 */
+      startTime: number;
+      /** @example 1666666666 */
+      endTime: number;
+      /** @example 1666666666 */
+      postedTime: number;
```

Comment:
Should this be inferred instead of supplied?

Reply:
Ah sorry yeah this shouldn't be in the create dto